language/stdlib/modules/AccountLimits.move

```diff
@@ -311,7 +316,7 @@ module AccountLimits {
         amount: u64,
         receiving: &mut Window<CoinType>,
     ): bool acquires LimitsDefinition {
-        let limits_definition = borrow_global_mut<LimitsDefinition<CoinType>>(receiving.limit_address);
+        let limits_definition = borrow_global<LimitsDefinition<CoinType>>(receiving.limit_address);
```

Comment:
This is good!  @tzakian and I also discovered in some part of the code that a mutable borrow was being taken when an immutable would suffice.

Reply:
This is potentially a candidate for a compiler/linter warning/error. @tnowacki